Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
The copy of rBoot included in the Sming merge is unmodified from the upstream version (ideally it will stay this way, I've made some tweaks for Sming in the upstream rather than make a it a fork). So why is it included upstream then? A good question, it could probably be taken out with a few tweaks to the Makefile, I'll have a look...